### PR TITLE
[7192] - Remove lead school UI from support console

### PR DESCRIPTION
**Pull request by kennyevil**
### Context

https://trello.com/c/VYZvyAli/7192-remove-lead-school-and-related-from-support-console

### Changes proposed in this pull request

Trying to go to the lead schools UI, either through the support console navigation, or directly in the browser, will no longer work.

### Important business

- [x] Does this PR introduce any PII fields that need to be overwritten or deleted in db/scripts/sanitise.sql?
- [x] Does this PR change the database schema? If so, have you updated the config/analytics.yml file and considered whether you need to send 'import_entity' events?
- [x] Do we need to send any updates to DQT as part of the work in this PR?
- [x] Does this PR need an ADR?

NB: Please notify the #twd_data_insights team and ask for a review if new fields are being added to analytics.yml

### CODE REVIEW

**Comment by defong** [comment]
We probably need to look into address this bit
image

image

File: app/controllers/system_admin/lead_schools/users_controller.rb [hunk 1]
```diff
@@ -1,30 +0,0 @@
...
-          redirect_to(lead_school_path(@lead_school), flash: { success: t(".success") })
-        else
-          render(:edit)
-        end
-      end
-
-    private
-
-      def set_lead_school
-        @lead_school = policy_scope(School, policy_scope_class: LeadSchoolPolicy::Scope).find(params[:lead_school_id])
```
Comment (Reyko) [inline app/controllers/system_admin/lead_schools/users_controller.rb:22]: Potentially the `LeadSchoolPolicy` could be removed as well if not used anywhere else.

**Review by Reyko — COMMENTED** [review]
Few more things to consider for removal:

1. [system_admin/lead_schools_controller.rb](https://github.com/DFE-Digital/register-trainee-teachers/blob/7192-remove-lead-school-ui-from-support-console/app/controllers/system_admin/lead_schools_controller.rb)
2. [system_admin/user_lead_schools_controller.rb](https://github.com/DFE-Digital/register-trainee-teachers/blob/7192-remove-lead-school-ui-from-support-console/app/controllers/system_admin/user_lead_schools_controller.rb)
3. [policies/lead_school_policy.rb](https://github.com/DFE-Digital/register-trainee-teachers/blob/7192-remove-lead-school-ui-from-support-console/app/policies/lead_school_policy.rb)

**Review by Reyko — APPROVED** [review]

**Comment by defong** [comment]
Closing as it is superseded by https://github.com/DFE-Digital/register-trainee-teachers/pull/4498

**Pull request closed without merging**